src/common/utility.ts

```diff
@@ -403,7 +403,8 @@ export const updateURLAsPerUserLanguage = () => {
     const paths = current_path.split('/')
     const first_path = paths[1]
     const has_language_in_url = first_path in language_config
-    has_language_in_url && Cookies.set('user_language', first_path)
+    document.cookie =
```

Comment:
By this approach I think we are creating a `cookie` without any name right?
If `has_language_in_url` would be `false` we will store a cookie with `false` value without any name.
Let's avoid this.